cmd/constants/providers.go

```diff
@@ -0,0 +1,142 @@
+	"github.com/skip-mev/slinky/providers/apis/marketmap"
+	"github.com/skip-mev/slinky/providers/volatile"
+	"github.com/skip-mev/slinky/providers/websockets/bitfinex"
+	"github.com/skip-mev/slinky/providers/websockets/bitstamp"
+	"github.com/skip-mev/slinky/providers/websockets/bybit"
+	"github.com/skip-mev/slinky/providers/websockets/coinbase"
+	"github.com/skip-mev/slinky/providers/websockets/cryptodotcom"
+	"github.com/skip-mev/slinky/providers/websockets/gate"
+	"github.com/skip-mev/slinky/providers/websockets/huobi"
+	"github.com/skip-mev/slinky/providers/websockets/kraken"
+	"github.com/skip-mev/slinky/providers/websockets/kucoin"
+	"github.com/skip-mev/slinky/providers/websockets/mexc"
+	"github.com/skip-mev/slinky/providers/websockets/okx"
+	mmtypes "github.com/skip-mev/slinky/service/clients/marketmap/types"
+)
+
+var (
+	Providers = []config.ProviderConfig{
+		{
+			Name: coinbaseapi.Name,
+			API:  coinbaseapi.DefaultAPIConfig,
+			Type: types.ConfigType,
+		},
+		{
+			Name: binanceapi.Name,
+			API:  binanceapi.DefaultNonUSAPIConfig,
+			Type: types.ConfigType,
+		},
+		{
+			Name: raydium.Name,
+			API:  raydium.DefaultAPIConfig,
+			Type: types.ConfigType,
+		},
+		{
+			Name: uniswapv3.ProviderNames[constants.ETHEREUM],
+			API:  uniswapv3.DefaultETHAPIConfig,
+			Type: types.ConfigType,
+		},
+		{
+			Name: krakenapi.Name,
+			API:  krakenapi.DefaultAPIConfig,
+			Type: types.ConfigType,
+		},
+		{
+			Name: marketmap.Name,
+			API:  marketmap.DefaultAPIConfig,
+			Type: mmtypes.ConfigType,
+		},
+		{
+			Name: volatile.Name,
+			API:  volatile.DefaultAPIConfig,
+			Type: types.ConfigType,
+		},
+		{
+			Name:      bitfinex.Name,
+			WebSocket: bitfinex.DefaultWebSocketConfig,
+			Type:      types.ConfigType,
+		},
+		{
+			Name:      bitstamp.Name,
+			WebSocket: bitstamp.DefaultWebSocketConfig,
+			Type:      types.ConfigType,
+		},
+		{
+			Name:      bybit.Name,
+			WebSocket: bybit.DefaultWebSocketConfig,
+			Type:      types.ConfigType,
+		},
+		{
+			Name:      coinbase.Name,
+			WebSocket: coinbase.DefaultWebSocketConfig,
+			Type:      types.ConfigType,
+		},
+		{
+			Name:      cryptodotcom.Name,
+			WebSocket: cryptodotcom.DefaultWebSocketConfig,
+			Type:      types.ConfigType,
+		},
+		{
+			Name:      gate.Name,
+			WebSocket: gate.DefaultWebSocketConfig,
+			Type:      types.ConfigType,
+		},
+		{
+			Name:      huobi.Name,
+			WebSocket: huobi.DefaultWebSocketConfig,
+			Type:      types.ConfigType,
+		},
+		{
+			Name:      kraken.Name,
+			WebSocket: kraken.DefaultWebSocketConfig,
+			Type:      types.ConfigType,
+		},
+		{
+			Name:      kucoin.Name,
+			WebSocket: kucoin.DefaultWebSocketConfig,
+			Type:      types.ConfigType,
+		},
+		{
+			Name:      mexc.Name,
+			WebSocket: mexc.DefaultWebSocketConfig,
+			Type:      types.ConfigType,
+		},
+		{
+			Name:      okx.Name,
+			WebSocket: okx.DefaultWebSocketConfig,
+			Type:      types.ConfigType,
+		},
+	}
+
+	MarketMapProviders = []config.ProviderConfig{
```

Comment:
`AlternativeMarketMapProviders` since the default one is already in the main config?